.eslintrc.js

```diff
@@ -243,7 +243,9 @@ module.exports = {
     // These are error by default for JS too
     '@typescript-eslint/no-unsafe-call': 'warn',
     '@typescript-eslint/no-unsafe-member-access': 'warn',
-    '@typescript-eslint/no-unsafe-return': 'warn',
+    // Turned off this rule since it was throwing
+    // TypeError: Cannot read property 'kind' of undefined
+    '@typescript-eslint/no-unsafe-return': 'off',
```

Comment:
@gitstart-sourcegraph, could you please create an issue in the repo to fix this problem?